src/status_im/ui/screens/wallet/request/views.cljs

```diff
@@ -61,17 +59,14 @@
         [react/view styles/network-container
          [react/view styles/qr-container
           [qr-code amount symbol]]]
-        [react/view wallet.styles/choose-wallet-container
-         [components/choose-wallet]]
         [react/view wallet.styles/amount-container
          [components/amount-input
           {:error         amount-error
            :input-options {:on-focus (fn [] (when @scroll (js/setTimeout #(.scrollToEnd @scroll) 100)))
-                           :on-change-text #(re-frame/dispatch [:wallet.request/set-and-validate-amount %])}}]
-         [react/view wallet.styles/choose-currency-container
-          [components/choose-currency {:style     wallet.styles/choose-currency
-                                       :on-change #(re-frame/dispatch [:wallet.request/set-symbol (keyword %)])
-                                       :value     (name (or symbol :ETH))}]]]]]
+                           :on-change-text #(re-frame/dispatch [:wallet.request/set-and-validate-amount %])}}]]
+        [react/view wallet.styles/choose-asset-container
+         [components/choose-asset {:type   :request
```

Comment:
Why is  :on-change #(re-frame/dispatch [:wallet.request/set-symbol (keyword %)]) no longer necessary?

Reply:
It's done when you select an element from the asset list now.